packages/ui-extensions/docs/surfaces/point-of-sale/reference/apis/order-api.doc.ts

```diff
@@ -25,6 +29,35 @@ The Order API provides an extension with data about the current order.
   ],
   category: 'APIs',
   related: [],
+  examples: {
+    description: 'Examples of using the Order API',
+    examples: [
+      {
+        codeblock: generateCodeBlockForOrderApi(
+          'Basic usage of the Order API with useOrder hook',
+          'basic-usage',
+        ),
+      },
+      {
+        codeblock: generateCodeBlockForOrderApi(
+          'Display order details in a custom block',
+          'display-in-block',
+        ),
+      },
+      {
+        codeblock: generateCodeBlockForOrderApi(
+          'Use Order API with navigation',
+          'with-navigation',
+        ),
+      },
+      {
+        codeblock: generateCodeBlockForOrderApi(
+          'Simple useOrder hook example',
+          'use-order-hook',
+        ),
+      },
```

Comment:
The navigation example is a hallucination as the extension can't navigate to Customer Details just yet. The Simple hook example is redundant with the Basic usage one.